modules/effects/arrow-effects-data/src/main/kotlin/arrow/effects/KleisliInstances.kt

```diff
@@ -0,0 +1,29 @@
+package arrow.effects
```

Comment:
@raulraja has told me a specific package is required for instances. Check with him before merging, please.